test/properties.test.js

```diff
@@ -1395,6 +1395,26 @@ describe("font", () => {
     testPropertyValue("font-family", "Hawaii 5-0, sans-serif", "");
   });
 
+  it("font-family should set / get family values", () => {
+    testPropertyValue("font-family", "generic(fangsong)", "generic(fangsong)");
+  });
+
+  it("font-family should set / get family values", () => {
+    testPropertyValue("font-family", "generic(kai)", "generic(kai)");
+  });
+
+  it("font-family should set / get family values", () => {
+    testPropertyValue("font-family", "generic(khmer-mul)", "generic(khmer-mul)");
+  });
+
+  it("font-family should set / get family values", () => {
+    testPropertyValue("font-family", "generic(nastaliq)", "generic(nastaliq)");
+  });
+
+  it("font-family should not set / get invalid family values", () => {
+    testPropertyValue("font-family", "generic(foo)", "");
```

Comment:
Is there spec support for this? Chrome and Firefox return back `generic(foo)` when I set it.

Reply:
Sorry, I made a typo while testing.